FormidableLabs/victory-docs
# Update favicon

## Summary
- Just updating the favicon for victory docs from the old logo to the new badge logo.

## To test
- Pull branch and build locally and serve
- You might have to clear your cache but you should see the updated icon on the browser tab now.

<img width="180" alt="Screen Shot 2020-03-02 at 1 40 36 PM" src="https://user-images.githubusercontent.com/10720454/75716025-ac59d780-5c8b-11ea-97e6-946b3d7a30cb.png">
